scripts/build_test.py

```diff
@@ -24,7 +33,79 @@
 class BuildTests(test_utils.GenericTestBase):
     """Test the build methods."""
 
+    def test_minify(self):
+        """Tests _minify with an invalid filepath."""
+        # pylint: disable=protected-access
+        with self.assertRaises(subprocess.CalledProcessError) as calledProcess:
+            build._minify(
+                'path/to/input.js', 'path/to/output.js')
+        # pylint: enable=protected-access
+        calledProcessException = calledProcess.exception
+        # returncode is the exit status of the child process.
+        self.assertEqual(calledProcessException.returncode, 1)
+
+    def test_join_files(self, file_count=10):
```

Comment:
Why pass in file_count? Nothing calls this method AFAICT.

Reply:
Setting file_count as variable down below!